app/src/main/scala/org/alephium/explorer/api/model/Event.scala

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer.api.model
+
+import scala.collection.immutable.ArraySeq
+
+import org.alephium.api.model.Val
+import org.alephium.explorer.api.Codecs._
+import org.alephium.json.Json._
+import org.alephium.protocol.model.{BlockHash, TransactionId}
+
+final case class Event(
+    blockHash: BlockHash,
+    txHash: TransactionId,
+    contractAddress: Address,
+    inputAddress: Option[Address],
+    eventIndex: Int,
```

Comment:
I put it as it's also in the [node api model](https://github.com/alephium/alephium/blob/master/api/src/main/scala/org/alephium/api/model/ContractEvents.scala#L39), we need them in the db to sort, but we can remove them from the api if you don't think it will be useful for anyone.

Reply:
Sorry, this field is very useful. Forgive my comments 😅